Simplenote/en.lproj/Localizable.strings

```diff
@@ -1,857 +0,0 @@
-/* Number of found search results */
```

Comment:
It seems surprising we would delete this entire file? I think something might've gone wrong here

Reply:
Fixed by #1437.